.github/workflows/release.yml

```diff
@@ -25,6 +25,14 @@ jobs:
       - name: Install build dependencies
         run: |
           python -m pip install build --user
+      - name: Build Substrait planloader library
+        run: |
+          cd ${{ github.workspace }}/third_party/substrait-cpp
+          make release
+      - name: Install Substrait planloader library
+        run: |
+          cd ${{ github.workspace }}/third_party/substrait-cpp/build-Release/export/planloader
+          make install
```

Comment:
We'll want to handle compilation via the `pyproject.toml` file, I'll leave some notes there.